README.md

```diff
@@ -2,47 +2,117 @@
 
 ## Partner Study
 #### Answers to the following partner-study can be found [here](https://reactjs.org/docs/glossary.html)
-  * Describe the following, Single Page Application, Compilers, Bundlers, Elements, Components, JSX, Package Mangers, CDN, Props and State. 
-  * Write all answers in answers.md
+  * Describe the following, writing all answer in `answers.md`
+    - Single Page Application,
+    - Compilers, 
+    - Bundlers,
+    - Elements,
+    - Components,
+    - JSX,
+    - Package Mangers,
+    - CDN, 
+    - Props and State.
 
 ### Other Useful Resources
 * [CodeSandbox](https://codesandbox.io/s/z667kplonx) that was used in the lecture video: 
 * [API Documentation](https://reactjs.org/docs/react-api.html)
 * [Documentation](https://facebook.github.io/react/docs/jsx-in-depth.html) on JSX
 
 ### Battle Plan
- * Objective: At this point you have become familiar with the DOM and have done some research on the wonderful world of HTML and CSS. Now we're going to dive into modern front-end JavaScript development by learning about ReactJS.
- * You're going to be building a ToDo App (please hold your applause).
- * We know this seems trivial, but the best part about this assignment is that is shows off some of the strengths of React and you can take it as far as want.
+ * **Objective:** 
+   - At this point  you have become familiar with the  DOM and have done
+     some research on the wonderful world of HTML and CSS.
+   - Now  we're   going  to  dive  into   modern  front-end   JavaScript
+     development by learning about ReactJS.
 
-  * Tool suggesstions
-    * React Dev Tools - *This is a MUST* you need to install this asap!
-    * We have everything you need for your React Developer environment in this file. We went over this in the lecture video.
+ * **Build:**
+   - You're going to be building a ToDo App (please hold your applause).
+   - We know this seems trivial, but the best part about this assignment
+     is that  is shows off  some of the strengths  of React and  you can
+     take it as far as want.
+
+  * **Tool suggesstions:**
+    * React Dev  Tools - *This is  a MUST* --- you need  to install this
+      asap!
+    * We have  everything you need for your  React Developer environment
+      in this file. We went over this in the lecture video.
 
 #### To Get Started
 ##### You'll need to make sure you have the following installed.
-  * node and npm
-  * `npm install` will pull in all the node_modules you need once you `cd` into the root directory of the project
-  * `npm start` will start a development server on your localhost at port 3000.
-  * `npm test` will run the tests that are included in the the project. Try to get as many of these passing as you can in the allotted time. 
+  * `node` and `npm`
+  * `yarn` will pull in all the node_modules you need once you `cd` into
+    the root directory of the project
```

Comment:
This sentence is kinda awkward. Maybe something like

 * `yarn`: run this command after you `cd` into the root directory of the project. The `yarn` command will install all the  node_modules for your project per the contents of the "package.json" file.